src/util/common-mixin.js

```diff
@@ -311,5 +311,26 @@ export default {
 
       return logos
     },
+    /**
+     *  Calculate readable category
+     *
+     * @param {*} packageRef
+     * @returns Returns a string value for category
+     */
+    getDeviceCategory(packageRef) {
+      let zclProperty = ''
+      if (this.$store.state.zap.isMultiConfig) {
+        zclProperty =
+          this.$store.state.zap.selectedZapConfig.zclProperties.find(
+            (item) => item.id === packageRef && item.category
+          )
+        return zclProperty.category
+      } else {
+        zclProperty = this.$store.state.zap.packages.find(
+          (item) => item.pkg.id === packageRef && item.pkg.category
+        )
+        return zclProperty.pkg?.category
```

Comment:
This return does not seem right. What happens if category does not exist(return pkg)?
One of the reasons to add cypress tests to cover these use cases.